pages/api/register.js

```diff
@@ -25,46 +25,10 @@ export default async function handler (request, response) {
         }
       })
       if (newUser) {
-        response.status(200).json({ message: 'New user created.' })
+        response.send(200).json({ message: 'New user created.' })
       } else {
-        response.status(406).json({ message: 'Could not create a new user.' })
+        response.send(417).json({ message: 'Could not create a new user.' })
       }
```

Comment:
you can simplify everything in this function to this:

```javascript
const { fullName, email, password } = request.body

await prisma.user.create({
        data: {
          companyId: 2,
          email: email,
          name: fullName,
          password: await bcrypt.hash(password, 10)
        }
 })

return response.send(200).json({ message: 'New user created.' })


```

Reply:
here a quick article on how to refactor nested if/else code smells:

https://dev.to/huzaifa99/2-simple-tips-to-clean-nested-if-else-conditions-2kn4

Reply:
Done